src/main/java/org/symphonyoss/symphony/messageml/markdown/nodes/OptionNode.java

```diff
@@ -0,0 +1,25 @@
+package org.symphonyoss.symphony.messageml.markdown.nodes;
+
+import org.commonmark.node.CustomNode;
+import org.commonmark.node.Delimited;
+import org.commonmark.node.Visitor;
+
+public class OptionNode extends CustomNode implements Delimited {
+  private final static String LEFT_DELIMITER = "-";
```

Comment:
Do you mean to have the delimiter without space (e.g. "-Option 1")? And do we want the delimiter to be a dash at all? A dash is also used for list items, so interpreting options as lists may or may not be what you want.

Reply:
As we talked on 1-1, this fits the Spec.